Automattic/wp-calypso
# Stats: Add layout styles for feedback card

Related to: https://github.com/Automattic/red-team/issues/155

## Proposed Changes

Adds basic layout styles to the feedback card. Should work correctly from mobile to Desktop screen sizes. Does not yet apply styles to the CTA text or buttons (including spacing and sizing of emoji).

<img width="909" alt="SCR-20240902-oorj" src="https://github.com/user-attachments/assets/386ccf80-8289-42ec-89c7-3715c152861e">

## Why are these changes being made?


Part of the User Feedback project.

pejTkB-1Ce-p2

## Testing Instructions



* Launch the Calypso Live site.
* Visit the Traffic page.
* Enable the feature flag: `?flags=stats/user-feedback`
* Confirm the feedback card is visible at the bottom of the page.
* Confirm it displays correctly when resizing the view.

## Pre-merge Checklist
